client/src/sim.rs

```diff
@@ -58,6 +58,10 @@ pub struct Sim {
     place_block_pressed: bool,
     /// Whether the break-block button has been pressed since the last step
     break_block_pressed: bool,
+
+    selected_material: Material,
+    material_palette: [Material; 10],
```

Comment:
nit: this might as well be a `const`, since we're not likely to mutate it any time soon.